node/src/BaseClient.ts

```diff
@@ -1316,6 +1317,31 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Gets the union of all the given sets.
```

Comment:
Same here - some docs start on the very first line of code comment, some - on the second.